forum_project/users/models.py

```diff
@@ -1,3 +1,60 @@
 from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from datetime import datetime
 
-# Create your models here.
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, password=None, **extra_fields):
+        # Ensure that email is provided and normalize it
+        if not email:
+            raise ValueError("The Email field must be set")
+        email = self.normalize_email(email)
+        
+        # Create the user without requiring a username field
+        user = self.model(email=email, **extra_fields)
+        
+        # Set password and registration date
+        user.set_password(password)
+        user.registration_date = datetime.now()  # You can customize this field as needed
```

Comment:
The line does not use Django's timezone-aware `timezone.now()`. Incorrect timestamps in databases with different time zones.
Use `from django.utils.timezone import now` and replace `datetime.now()` with `timezone.now()`.